### PR TITLE
View Unit Test

### PR DESCRIPTION
**Pull request by sebworks**
View Unit Test - Using Ampersand-View (https://ampersandjs.com/learn/view-conventions ) conventions and Angular (https://github.com/johnpapa/angular-styleguide#naming) naming conventions to standardize View implementations.

## Changes
- Updated `test/unit_tests/.eslintrc` to add exception for `no-new`.
- Added `test/unit_tests/fixtures/View.js` to use as a baseline and describe View conventions.
- Added `test/unit_tests/modules/util/view-spec.js` to test all View implementations.

## Notes
- ~~Working on getting test to run on `*View.js` ( any js file ending in `View.js` )~~. 
- Update: Added in (https://github.com/cfpb/cfgov-refresh/commit/9e2fbe37a0105b5b8cf1cbe011c5ec2d7b3e5066)

## Testing
- Run `gulp test:unit:js`

## Review
@anselmbradford 
@jimmynotjim
@KimberlyMunoz

### CODE REVIEW

**Comment by KimberlyMunoz** [comment]
I talked with @sebworks about this and #905 and it seems like an solid way to standardize and structure our JS in a testable way. View.js is a template for what other "view" modules will be modeled after.

**Comment by sebworks** [comment]
thanks @KimberlyMunoz

**Comment by sebworks** [comment]
Closing this PR per https://github.com/cfpb/cfgov-refresh/pull/905#issuecomment-132827812.

File: test/unit_tests/modules/view-spec.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    beforeEach( function() {
+      viewInstance = new View();
+      viewInstancewithEl = new View( { el: document.createElement( 'div' ) } );
+    } );
+
+    context( 'has a constructor and it', function() {
+      it( 'should be passed an object', function() {
+        var constructorSpy = sinon.spy( exports, 'View' );
+        View = exports.View;
+        new View( {} ); // eslint-disable-line max-statements, no-inline-comments, max-len
```
Comment (anselmbradford) [inline test/unit_tests/modules/view-spec.js]: This is `max-statements`? It looks like `no-new`

File: test/unit_tests/modules/view-spec.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        var arg0 = constructorSpy.args && constructorSpy.args[0];
+        expect( arg0 !== null && typeof arg0 === 'object' ).to.be.true;
```
Comment (anselmbradford) [inline test/unit_tests/modules/view-spec.js]: Does this actually test that it has a constructor? Maybe that's implied through the use of arguments, but it seems like this is just checking that there's an argument and it's an object.

File: test/unit_tests/modules/view-spec.js (hunk 1 above)
Comment (sebworks) [inline test/unit_tests/modules/view-spec.js]: yep, the comment can be removed as its being set in the `.eslintrc` file.

File: test/unit_tests/modules/view-spec.js (hunk 2 above)
Comment (sebworks) [inline test/unit_tests/modules/view-spec.js]: yep, it's just implied because the function was invoked using the `new` keyword.

**Pull request closed without merging**